CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT Ruler: Added the following metrics: #2786
```

Comment:
CHANGELOG entries are grouped by type. Could you move this entry below, along with the other `[ENHANCEMENT]` entries, please?